src/style.css

```diff
@@ -0,0 +1,32 @@
+html {
```

Comment:
It's better to apply the margin reset to the 'body' element, not the 'html' element.